### PR TITLE
Set kafka-webhook Deployments (indirectly) via HPA

### PR DESCRIPTION
**Pull request by lberk**
Related: SRVKE-884

### CODE REVIEW

**Comment by lberk** [comment]
@matzew as we chatted about -- first hack at this for kafka

File: knative-operator/pkg/controller/knativekafka/knativekafka_controller.go [hunk 1]
```diff
@@ -465,6 +465,17 @@ func setKafkaDeployments(replicas int32) mf.Transformer {
...
 			}
+		} else if u.GetKind() == "HorizontalPodAutoscaler" {
+			min, _, err := unstructured.NestedInt64(u.Object(), "spec", "minReplicas")
+			if err != nil {
+				return err
+			}
+			if min > replicas {
+				return nil
+			}
+			if err := unstructured.SetNestedField(u.Object(), replicas, "spec", "minReplicas"); err != nil {
```
Comment (matzew) [inline knative-operator/pkg/controller/knativekafka/knativekafka_controller.go:476]: should `replicas` be ->   `int64(replicas)` ?

Comment (lberk) [reply knative-operator/pkg/controller/knativekafka/knativekafka_controller.go]: yep, fixed the build errors and rebased :+1:

**Pull request merged**